locale/gd.js

```diff
@@ -0,0 +1,85 @@
+//! moment.js locale configuration
```

Comment:
moment has a very strange build/release process that means more files than are sensible are maintained in version control